smarts/core/vehicle.py

```diff
@@ -149,25 +147,15 @@ def __init__(
             self._color = config.color
 
         # TODO: Move this into the VehicleGeometry class
-        self._np = self._build_model(pose, config, showbase)
+        self._renderer_path = renderer.create_vehicle(
```

Comment:
I might consider naming this `_renderable`, renderer has the meaning of being the driver of rendering rather than an item to be rendered.

Reply:
Yes, I agree.  That's better.  Will do.

Reply:
Actually, there's no need since `renderer_path` has been removed completely.